### PR TITLE
Better structure for project's tests

### PR DESCRIPTION
**Pull request by rpunkfu**

### CODE REVIEW

File: test/index.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
-import { find, load, all, exist, getUrl, parse } from './gh-emoji';
-import fixtures from './fixtures';
+import { find, load, all, exist, getUrl, parse } from '../gh-emoji';
```
Comment (ryyppy) [inline test/index.js:1]: Next logical step for me would be to put the implementation code to `./src` :-)

**Comment by ryyppy** [comment]
Looking good! Good job!

File: test/index.js (hunk 1 above)
Comment (rpunkfu) [inline test/index.js:1]: Yes, but in separate PR right? :)

**Comment by rpunkfu** [comment]
Do you think it's ready to merge @ryyppy?

**Pull request merged**